app/src/main/java/cm/aptoide/pt/smart/appfiltering/data/Rating.java

```diff
@@ -10,17 +10,17 @@
 public class Rating {
 
     @SerializedName("avg")
-    private Long mAvg;
+    private Double mAvg;
```

Comment:
For a version of Teams I was testing, this field was a floating point value. There's a similar class called "PRating" that already has mAvg set to a "Double", so I just used that here.